cypress/integration/examples/assertions.spec.js

```diff
@@ -124,6 +124,36 @@ context('Assertions', () => {
         })
     })
 
+    it('matches unknown text between two elements', () => {
+      /**
+       * Text from the first element.
+       * @type {string}
+      */
+      let text
```

Comment:
I don't think we should be introducing TypeScript into this repo's example code or tests. This is supposed to support super beginners to our project. I wouldn't assume them to have any TypeScript knowledge.

Reply:
😯